lib/App/DH.pm

```diff
@@ -261,15 +261,31 @@ sub cmd_upgrade {
   return;
 }
 
+=cmd version
+
+Report version of connection L</--connection_name>
```

Comment:
It should be noted that I don't think `->schema_version` queries the database to get the current database schema version.

I believe it does nothing more than query the logic in the relevant perl modules for the version of the *modules* schema.

Hence why `write_ddl` uses that value as the "target version"

Perhaps this is grounds for a different set of commands, like `current_version` and `target_version` ?

However, implementing `current_version` is probably a lot of fun, and I'm kinda unsure what use `target_version` is.